readthedocs/projects/forms.py

```diff
@@ -681,17 +678,7 @@ class Meta:
             ),
             "notifications_show_on_latest": _("Show a notification on latest version"),
             "linkpreviews_enabled": _("Enabled"),
-            "linkpreviews_root_selector": _("Root selector"),
-            "linkpreviews_doctool_name": _("Documentation tool name"),
-            "linkpreviews_doctool_version": _("Documentation tool version"),
-        }
-        widgets = {
-            "doc_diff_root_selector": forms.TextInput(
-                attrs={"placeholder": AddonsConfig.DOC_DIFF_DEFAULT_ROOT_SELECTOR}
```

Comment:
Would be nice to have a placeholder still in the `options_root_selector` I think? So that folks have at least a sense of what they can put in.